pytorch_lightning/profiler/pytorch.py

```diff
@@ -102,202 +140,207 @@ def __init__(
                     nvvp trace_name.prof
                     torch.autograd.profiler.load_nvprof(path)
 
-            export_to_chrome: Wether to export the sequence of profiled operators for Chrome.
+            export_to_chrome: Whether to export the sequence of profiled operators for Chrome.
                 It will generate a ``.json`` file which can be read by Chrome.
 
             path_to_export_trace: Directory path to export ``.json`` traces when using ``export_to_chrome=True``.
                 By default, it will be save where the file being is being run.
 
-            row_limit: Limit the number of rows in a table, `0` is a special value that
+            row_limit: Limit the number of rows in a table, ``0`` is a special value that
                 removes the limit completely.
```

Comment:
why not `-1` ? i'd expect 0 to mean keep no rows

Reply:
Good catch !